app/assets/javascripts/angular/angular-config.js.erb

```diff
@@ -7,3 +7,9 @@ app.config(['$routeProvider',function($routeProvider){
   })
 }])
 
+app.config(['$routeProvider',function($routeProvider){
+  $routeProvider.when('/pin/', {
+    templateUrl: "<%= asset_path('pin-show.html') %>",
+    controller: 'PinCtrl',
```

Comment:
I dont need the controller to see the page. We're not using the controller until we have the scrape data